src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -52,6 +57,34 @@ export default class EthereumBlockchainService implements BlockchainService {
     this._chainId = BASE_CHAIN_ID + ':' + idnum
   }
 
+  /**
+   * Sets the gas price for the transaction request
+   * @param txData - transaction request data
+   * @param attempt - what number attempt this is at submitting the transaction.  We increase
+   *   the gas price we set by a 10% multiple with each subsequent attempt
+   * @private
+   */
+  private async _setGasPrice(txData: TransactionRequest, attempt: number): Promise<void> {
+    const { overrideGasConfig } = config.blockchain.connectors.ethereum;
+    if (config.blockchain.connectors.ethereum.overrideGasConfig) {
+      txData.gasPrice = BigNumber.from(config.blockchain.connectors.ethereum.gasPrice);
+      logger.debug('Overriding Gas price: ' + txData.gasPrice.toString());
+
+      txData.gasLimit = BigNumber.from(config.blockchain.connectors.ethereum.gasLimit);
+      logger.debug('Overriding Gas limit: ' + txData.gasLimit.toString());
+    } else {
+      const gasPriceEstimate = await this.provider.getGasPrice();
+      // Add 10% extra to gas price for each subsequent attempt
+      txData.gasPrice = gasPriceEstimate.add(gasPriceEstimate.mul(attempt * .01));
```

Comment:
Related to comment below. This makes sense to bump gas price if the problem is actually gas.